website/docs/intro.md

```diff
@@ -2,7 +2,18 @@
 sidebar_position: 1
 ---
 
-# Introduction
-
-> Work in progress...
+# General
 
+> About plugin
+What is ECInventory for? It is a powerful tool for changing your inventory to your liking. With it, you can make your server even more atmospheric. You will be able to add a lot of new highly customizable inventories. Thanks to the Mimic API, ECInventory can use the capabilities of other plugins (MMOItems, MMOCore, QuantumRPG, SkillAPI, PROSkillAPI, Heroes, etc.).
```

Comment:
Quote (`>`) is not needed here. It's better to split paragraph by sentences to make Git diffs cleaner when we change one of sentences:
```suggestion
## What is ECInventory for?

It is a powerful tool for changing your inventory to your liking.
With it, you can make your server even more atmospheric.
You will be able to add a lot of new highly customizable inventories.
Thanks to the Mimic API, ECInventory can use the capabilities of other plugins (MMOItems, MMOCore, QuantumRPG, SkillAPI, PROSkillAPI, Heroes, etc.).
```